spec/presenters/hyrax/etd_presenter_spec.rb

```diff
@@ -7,10 +7,27 @@
   let(:etd)           { FactoryGirl.create(:moomins_thesis, id: 'moomin_id') }
   let(:request)       { instance_double('Rack::Request', host: 'example.com') }
 
+  let(:attributes) do
```

Comment:
These are no longer used, and should just be dropped. This is currently re-adding them.